dags/slack_message.py

```diff
@@ -6,6 +6,7 @@ def slack_message(msg, notification=False, broadcast=False, attachment=None):
     try:
         slack_workername = BaseHook.get_connection(SLACK_CONN_ID).login
         slack_token = BaseHook.get_connection(SLACK_CONN_ID).password
+        notification_channel = BaseHook.get_connection(SLACK_CONN_ID).host
```

Comment:
Same question here. It seems helpful to add comments around these uses if this is the right thing to do.